src/main/java/pl/pop/interview/master/account/Account.java

```diff
@@ -4,13 +4,13 @@
 import lombok.*;
 import pl.pop.interview.master.practitioner.Practitioner;
 
-@Entity
-@Getter
-@Setter
+@Data
```

Comment:
hej, niestety adnotacja @Data jest niepożądana. Może powodować problemy przy porównywaniu obiektów bo nadpisuje hascode i equalsa. Lepiej wrócić do osobnych adnotacji, tylko tych ktróych potrzebujemy

Reply:
https://dzone.com/articles/lombok-and-jpa-what-may-go-wrong